lib/main.dart

```diff
@@ -101,7 +109,7 @@ class _MyHomePageState extends State<MyHomePage> {
                       );
                     },
                   )
-                : DirectoryGridView(children: cards);
+                : DirectoryGridView(children: cardBox.get('cards'));
```

Comment:
It is not a good practice to pass a db object. It would be better to create a Dart object that reads the hive object, then pass it to a view. Here you can see great examples in the [docs](https://docs.hivedb.dev/#/custom-objects/hive_object)

Also, we would need to do the same thing for the Details page.